docs/content/users/quickstart.md

```diff
@@ -397,6 +397,37 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev launch
     ```
 
+=== "Laravel Installer"
+
+    ```bash
+    mkdir my-laravel-site && cd my-laravel-site
+    ddev config --project-type=laravel --docroot=public
+
+    # Temporarily add the Laravel installer
+    echo 'ARG COMPOSER_HOME=/usr/local/composer
+    RUN composer global require laravel/installer
+    RUN ln -s $COMPOSER_HOME/vendor/bin/laravel /usr/local/bin/laravel
+    ' > .ddev/web-build/Dockerfile.laravel
+
+    # Start the project
+    ddev start
+
+    # Select a starter kit of your choice.
+    # The database is temporarily set to SQLite and will be switched to MariaDB
+    ddev exec laravel new temp --database=sqlite
+
+    # Move the installed files
+    ddev exec 'rsync -rltgopD temp/ ./ && rm -rf temp'
+
+    # Remove the Laravel installer and the .env file
+    ddev exec 'rm -rf .ddev/web-build/Dockerfile.laravel .env'
```

Comment:
Why not do this on the host? (not that it matters) Expecting `rm` not to be available? `rm` even works in PowerShell, but not Cmd.

Reply:
Okay, let's use the host.